ingests_tracker/src/main/resources/application.conf

```
@@ -0,0 +1 @@
+aws.dynamo.tableName=${?archive_ingest_table_name}
```

What we're using elsewhere:

```suggestion
aws.dynamo.tableName=${?ingests_table_name}
```